gabbi/runner.py

```diff
@@ -243,6 +253,13 @@ def _make_argparser():
         help='Custom response handler. Should be an import path of the '
              'form package.module or package.module:class.'
     )
+    parser.add_argument(
+        '-l', '--local-handlers',
+        default=False,
+        dest='local_handlers',
+        action='store_true',
+        help='Response handlers are relative to the current directory.'
```

Comment:
are -> may be

As the change you've done is making it so the can be loaded locally, but they might also be loaded from somewhere else if they are not found on .

Reply:
Updated.